website/source/docs/guides/servers.html.md

```diff
@@ -18,28 +18,67 @@ That means if there are 3 server nodes, at least 2 must be available.
 In general, if you are ever adding and removing nodes simultaneously, it is better
 to first add the new nodes and then remove the old nodes.
 
-## Adding New Servers
+In this guide, we will cover the different methods for adding and removing servers.
 
-Adding new servers is generally straightforward. Simply start the new
+## Manually Add a New Server
+
+Manually adding new servers is generally straightforward. Simply start the new
 agent with the `-server` flag. At this point, the server will not be a member of
 any cluster, and should emit something like:
 
-```text
+```sh
+consul agent -server
 [WARN] raft: EnableSingleNode disabled, and no known peers. Aborting election.
 ```
 
 This means that it does not know about any peers and is not configured to elect itself.
 This is expected, and we can now add this node to the existing cluster using `join`.
 From the new server, we can join any member of the existing cluster:
 
-```text
-$ consul join <Node Address>
+```sh
+$ consul join <Existing Node Address>
 Successfully joined cluster by contacting 1 nodes.
 ```
 
 It is important to note that any node, including a non-server may be specified for
-join. The gossip protocol is used to properly discover all the nodes in the cluster.
-Once the node has joined, the existing cluster leader should log something like:
+join. Generally, this method is good for testing purposes but not recommended for production
+deployments. For production clusters, you will likely want to use the agent configuration
+option to add additional servers.
+
+## Add a Server with Agent Configuration
+
+When adding addtional servers in production, you will likely want to use the
+[agent configuiration](https://www.consul.io/docs/agent/options.html)
+option, either with the Consul CLI or in the agent configuration file. With either
+option, you can use the `retry_join` setting.
+
+With the Consul CLI:
+
+```sh
+$ consul agent -retry-join=["52.10.110.11", "52.10.110.12", "52.10.100.13"]
+```
+
+In the agent configuration file:
+
+```sh
+{
+  "bootstrap": false,
+  "bootstrap_expect": 3,
+  "server": true,
+  "retry_join": ["52.10.110.11", "52.10.110.12", "52.10.100.13"]
+}
+```
+
+The `retry_join` will also ensure that if any server loses connection
+with the cluster for any reason, including the node restarting, it can
+rejoin when it comes back. Both servers and clients can use this method.
+
+### Server Coordination
+
+To ensure Consul servers are joining the cluster properly, you should monitor
+the server coordiantion. The gossip protocol is used to properly discover all
```

Comment:
typo: coordination

Reply:
Thanks for reviewing and catching those typos!